mne/epochs.py

```diff
@@ -380,14 +381,12 @@ class BaseEpochs(ProjMixin, ContainsMixin, UpdateChannelsMixin, ShiftTimeMixin,
     def __init__(self, info, data, events, event_id=None, tmin=-0.2, tmax=0.5,
                  baseline=(None, 0), raw=None, picks=None, reject=None,
                  flat=None, decim=1, reject_tmin=None, reject_tmax=None,
-                 detrend=None, proj=True, on_missing='error',
+                 detrend=None, proj=True, on_missing='raise',
```

Comment:
this is a silent API change. I would suggest to keep it as it is here but let the function interpret 'error' as 'raise' and same for warning or warn as I don't want to touch API of Epochs object. Here you are breaking backward compat.

Reply:
you actually accept error in the _on_missing function so don't change anything to Epochs API

Reply:
I prefer to standardize around raise, see discussion above

This won't break backward compatibility, we leave error as an alias, no need to remove it (almost no effort required to support the old error/warning and raise/warn)